sessions-core/src/main/java/com/adyen/checkout/sessions/core/CheckoutSessionProvider.kt

```diff
@@ -8,31 +8,29 @@
 
 package com.adyen.checkout.sessions.core
 
-import com.adyen.checkout.components.core.internal.Configuration
 import com.adyen.checkout.components.core.Order
+import com.adyen.checkout.components.core.internal.Configuration
 import com.adyen.checkout.sessions.core.internal.CheckoutSessionInitializer
-import kotlinx.coroutines.CoroutineScope
 
-// TODO docs
 object CheckoutSessionProvider {
-    // TODO docs
+
+    /**
+     * Allows creating a [CheckoutSession] from the response of the /sessions endpoint.
+     * This is a suspend function that executes a network call. Make sure you run this in a background thread.
```

Comment:
Should we use `withContext` to prevent mistakes?

Reply:
Can you check again?